app/views/shared/metadata/edit/_study.html.erb

```diff
@@ -46,9 +46,11 @@
         <%= metadata_fields.text_field(:ega_dac_accession_number) %>
         <%= metadata_fields.text_field(:ega_policy_accession_number) %>
         <%= metadata_fields.text_field(:array_express_accession_number) if study.for_array_express? %>
-        <%= metadata_fields.text_field :hmdmc_approval_number %>
+        <%= metadata_fields.text_field :hmdmc_approval_number unless study.new_record? %>
       <% end %>
 
+      <%= metadata_fields.text_field :hmdmc_approval_number if study.new_record? %>
```

Comment:
Sorry I don't know this study page very well but how come it needs to be defined in two places?

Reply:
The change was requested for new study creation but the template is also used by editing and managing the study. 
- If it is an existing study record, we continue using the existing behaviour, as part of Managed options.
- If it is a new study record, we show the approval number field for all three data release strategy options. We take it out from the other block so it is common to all three.

Reply:
We are changing this in the later commits. The field will be available for creating, editing, and managing study. The approval number will be available in all three data release strategy options while creating or updating the study.